packages/hdwallet-core/src/utils.ts

```diff
@@ -208,3 +208,7 @@ export function compatibleBufferConcat(list: Uint8Array[]): Buffer {
   if (!checkBufferConcat()) return Buffer.concat(list);
   return Buffer.concat(list.map(x => Buffer.isBuffer(x) ? x : Buffer.from(x)));
 }
+
+export function isIndexable(x: unknown): x is Record<string, unknown> {
```

Comment:
1. I do not like the name. It took some time to guess what it was for
2. No JSDoc block with an explanation of what the function does. For a utility function with a specific purpose, a JSDoc comment would be incredibly useful.

Reply:
1) I agree, but I can't think of a better one
2) Good point, agree, will update

Reply:
Now with 100% more JSDoc!